readme.md

```diff
@@ -29,14 +29,46 @@ Openwisp wifi login pages app to allow users to authenticate, sign up and know m
 
 ### Install
 
-##### Clone this repo
+#### Install openwisp-radius
```

Comment:
@NoumbissiValere Is it possible to give the user the installation instructions from [openwisp-radius docs](https://openwisp-radius.readthedocs.io/en/latest/developer/setup.html#installing-for-development)?

This will enable our docs to be more DRY, i.e: when a change is made in installation process, we would not need to update here as well.

Also, that documentation is more verbose! :smile:

What do you think?

Reply:
You are very correct @atb00ker :+1: